pom.xml

```diff
@@ -238,6 +253,18 @@
       <artifactId>hashicorp-vault-plugin</artifactId>
       <optional>true</optional>
     </dependency>
+    <!-- X.509 client certificate credentials -->
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>docker-commons</artifactId>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <!-- Marked as optional in docker-commons plugin, specify explicitly to update required version -->
+      <groupId>org.jenkinsci.plugins</groupId>
+      <artifactId>pipeline-model-extensions</artifactId>
+      <optional>true</optional>
+    </dependency>
```

Comment:
should be unneeded - the version is specified in `dependencyManagement` which will manage trasnitive dependnecy versions. 
```suggestion
```